config/types.go

```diff
@@ -44,19 +44,19 @@ type ActionRedisConfig struct {
 }
 
 type Client struct {
-	Network            string        `json:"network" jsonschema:"enum=tcp,enum=udp,enum=unix"`
-	Address            string        `json:"address"`
-	TCPKeepAlive       bool          `json:"tcpKeepAlive"`
-	TCPKeepAlivePeriod time.Duration `json:"tcpKeepAlivePeriod" jsonschema:"oneof_type=string;integer"`
-	ReceiveChunkSize   int           `json:"receiveChunkSize"`
-	ReceiveDeadline    time.Duration `json:"receiveDeadline" jsonschema:"oneof_type=string;integer"`
-	ReceiveTimeout     time.Duration `json:"receiveTimeout" jsonschema:"oneof_type=string;integer"`
-	SendDeadline       time.Duration `json:"sendDeadline" jsonschema:"oneof_type=string;integer"`
-	DialTimeout        time.Duration `json:"dialTimeout" jsonschema:"oneof_type=string;integer"`
-	Retries            int           `json:"retries"`
-	Backoff            time.Duration `json:"backoff" jsonschema:"oneof_type=string;integer"`
-	BackoffMultiplier  float64       `json:"backoffMultiplier"`
-	DisableBackoffCaps bool          `json:"disableBackoffCaps"`
+	Network            string        `json:"network" jsonschema:"enum=tcp,enum=udp,enum=unix" yaml:"network"`
```

Comment:
Why did you add `yaml:"..."` struct tag?

Reply:
It seems that `github.com/knadh/koanf/parsers/yaml` used in `configs.go` lowers the names in the YAML structure for nested maps if the YAML tags are not specified in the struct.